packages/appkit/src/networks/index.ts

```diff
@@ -2,5 +2,8 @@
 export * from 'viem/chains'
 export * from './solana/index.js'
 
+// -- Utils ------------------------------------------------------------------
+export * from './utils.js'
```

Comment:
can we export only what's needed? or is it all needed?

Reply:
Since there is only one module + since it's easier to not miss new added utils, this would make sense to export from utils file directly